helm_deploy/values-staging.yaml

```diff
@@ -14,6 +14,7 @@ generic-service:
     SPRING_PROFILES_ACTIVE: "stdout"
     HMPPS_AUTH_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
     PRISON_API_URL: https://api-dev.prison.service.justice.gov.uk
+    PRISONER-OFFENDER-SEARCH_URL: https://prisoner-offender-search-dev.prison.service.justice.gov.uk
```

Comment:
as above

Reply:
updated to PRISONER_OFFENDER_SEARCH_URL.